views/layout.php

```diff
@@ -51,35 +48,39 @@
         <body>
             <!-- Mise en place de la topbar -->
             <?php
-                include 'views/'.$topbar.'.php';
+                include ($topbar);
             ?>
 
             <!-- Mise en place du contenu -->
             <div class="container">
                 <?php
-                    include 'views/'.$page.'.php';
+                    echo $contentPage;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of echo language construct is discouraged.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)